### PR TITLE
feat(StatusCard): Create a new reusbale card component

### LINKED ISSUE #582: Networks Component
This Components need to:

- Support an API in which an array of supported networks is passed as a parameter, as well as a mapping of what networks connect to what destination networks
- the parameters should include flags for what networks are enabled, disabled, or hidden. and flags for display fields and/or values

Screen Shot 2022-03-17 at 3 25 14 PM
Screen Shot 2022-03-17 at 3 25 17 PM
Screen Shot 2022-03-17 at 3 25 26 PM

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
closes #582

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [x] add documentation if necessary (new component, new feature)
- [x] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [x] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [x] test changes in [status-desktop](https://github.com/status-im/status-desktop)

image

### CODE REVIEW

File: src/StatusQ/Components/StatusCard.qml [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+        }
+   \endqml
+   For a list of components available see StatusQ.
+*/
+
+Rectangle {
+    id: statusCard
+
+    /*!
+       \qmlproperty bool StatusCard::disabledText
```
Comment (caybro) [inline src/StatusQ/Components/StatusCard.qml:38]: `\qmlproperty string ...`

File: src/StatusQ/Components/StatusCard.qml [hunk 2]
```diff
@@ -0,0 +1,478 @@
...
+    */
+    property alias errorIconName: errorIcon.icon
+    /*!
+       \qmlproperty string StatusCard::cardIconName
+       Used to assign an icon to the card icon in StatusCard
+    */
+    property alias cardIconName: cardIcon.icon
+
+    /*!
+       \qmlproperty Text StatusCard::primaryLabel
```
Comment (caybro) [inline src/StatusQ/Components/StatusCard.qml:91]: `\qmlproperty alias ~`

Same below

File: src/StatusQ/Components/StatusCard.qml [hunk 3]
```diff
@@ -0,0 +1,498 @@
...
+            tertiaryText: "BALANCE: " + 250
+            cardIconName: "status"
+            advancedMode: false
+        }
+   \endqml
+   For a list of components available see StatusQ.
+*/
+
+Rectangle {
+    id: statusCard
```
Comment (caybro) [inline src/StatusQ/Components/StatusCard.qml:35]: "root" please :)

File: src/StatusQ/Core/Utils/Utils.qml [hunk 4]
```diff
@@ -75,6 +75,73 @@ QtObject {
...
+
+    // function to draw arrow
+    function drawArrow(context, fromx, fromy, tox, toy, color) {
+        const dx = tox - fromx;
+        const dy = toy - fromy;
+        const headlen = 10; // length of head in pixels
+        const angle = 0
+        const radius = 5
+
+        context.strokeStyle = color ? color : '#627EEA'
```
Comment (caybro) [inline src/StatusQ/Core/Utils/Utils.qml:87]: Shouldn't that color come from a Theme? Will it always be the same for light/dark?

**Review by caybro — CHANGES_REQUESTED** [review]
Just 2 minor things, otherwise looks good. Sorry for not noticing earlier :)

**Review by alexandraB99 — APPROVED** [review]
excelent, lgtm! just the comments from @caybro :)

**Review by caybro — APPROVED** [review]

**Pull request merged**